atoms.js

```diff
@@ -37,25 +37,25 @@ function AtomRW(width, readAtomFrom, writeAtomInto) {
 }
 inherits(AtomRW, BufferRW);
 
-AtomRW.prototype.byteLength = function byteLength() {
-    return new LengthResult(null, this.width);
+AtomRW.prototype.poolByteLength = function byteLength(destResult) {
```

Comment:
AtomRW.prototype.byteLength et al are public interfaces that have been removed. These *are* used by ThriftRW directly.